src/main/java/de/tum/in/www1/artemis/repository/SelfLearningFeedbackRequestRepository.java

```diff
@@ -0,0 +1,18 @@
+package de.tum.in.www1.artemis.repository;
+
+import static de.tum.in.www1.artemis.config.Constants.PROFILE_CORE;
+
+import org.springframework.context.annotation.Profile;
+import org.springframework.data.jpa.repository.JpaRepository;
+import org.springframework.stereotype.Repository;
+
+import de.tum.in.www1.artemis.domain.SelfLearningFeedbackRequest;
+
+/**
+ * Spring Data JPA repository for the SelfLearningFeedbackRequest entity.
+ */
+@Profile(PROFILE_CORE)
+@Repository
+public interface SelfLearningFeedbackRequestRepository extends JpaRepository<SelfLearningFeedbackRequest, Long> {
```

Comment:
```suggestion
public interface SelfLearningFeedbackRequestRepository extends ArtemisJpaRepository<SelfLearningFeedbackRequest, Long> {
```
We now have a new super interface for our repositories